src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+.cards {
+  display: grid;
+  grid-gap: 46px 48px;
+  padding: 50px $space-40;
+}
+
+.card {
+  margin-bottom: 0;
+  width: 200px;
+
+  background-color: $button-light-color;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+}
+
+.card > img {
+  width: 160px;
+  display: block;
+  margin: 32px auto 0;
+}
+
+.card__details {
+  margin: $space-40 16px 0;
+}
+
+.card__details-title {
+  font-size: $font-size-12;
+  font-weight: $font-weight-500;
+  line-height: $line-height-18;
+  color: $main-color;
+  margin: 0;
+
+  &:hover {
+    color: $title-hover-color;
+    transition-duration: $duration-time;
+  }
+}
+
+.card__details-code {
+  margin: 0;
+  padding-top: 5px;
+  font-size: 10px;
+
+  line-height: $line-height-14;
+  color: $secondary-color;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px;
+}
+
+.card__price-text {
+  font-size: $font-size-12;
+  margin: 0;
+
+  line-height: $line-height-18;
+  color: $secondary-color;
+}
+
+.card__price-number {
+  font-size: 16px;
+  font-weight: $font-weight-700;
+  line-height: $line-height-18;
+  color: $main-color;
+  margin: 0;
+}
+
+.card__button a {
```

Comment:
same here. add a class for a tag. and I believe you don't need an additional wrapper for your button. You can delete card__button div and move class to a directly

Reply:
Hi, thanks, same here - that was my original idea, but unfortunately it's not possible to have more than 2 classes in 1 element.